app/managers.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any
+
+from .display import DisplayConsole, DisplayReverse
+from .print import PrintConsole, PrintReverse
+from .serializer import SerializeJson, SerializeXml
+
+
+class DisplayManager:
+    display_managers = {}
```

Comment:
OCP violation in all managers you're using dict structure, if you want to implement managing system you should create dataclass which will be handled by respective manager or change the logic overall. I suggest you to remove managers logic for now and dive deeper into it )